src/schema/v2/partners.ts

```diff
@@ -139,17 +139,32 @@ export const Partners: GraphQLFieldConfig<void, ResolverContext> = {
 export const PartnersConnection: GraphQLFieldConfig<void, ResolverContext> = {
   type: connectionWithCursorInfo({ nodeType: PartnerType }).connectionType,
   description: "A list of Partners",
-  args: pageable({
-    ids: {
-      type: new GraphQLList(GraphQLString),
-    },
-  }),
+  args: pageable(
+    Object.assign(
+      {
+        ids: {
+          type: new GraphQLList(GraphQLString),
+        },
+      },
+      pick(
+        Partners.args,
+        "near",
+        "eligibleForListing",
+        "defaultProfilePublic",
+        "sort"
+      )
+    )
+  ),
   resolve: (_root, { ..._options }, { partnersLoader }) => {
```

Comment:
#minor but usually `_` is reserved for when arguments are unused. Can we make this `options` instead?

Reply:
Fixed